fpdf/fpdf.py

```diff
@@ -296,6 +296,7 @@ def __init__(
         self._toc_placeholder = None  # ToCPlaceholder
         self._outline = []  # list of OutlineSection
         self.section_title_styles = {}  # level -> TitleStyle
+        self.total_pages = 0
```

Comment:
No need to introduce a new attribute:
this information can be retrieved through `len(self.pages)`

Reply:
Not that you could add a new property-method as part of this PR if you want:
```python
@property
def pages_count(self):
    return len(self.pages)
```